deployments/sandbox/environment.py

```diff
@@ -55,17 +55,17 @@ def env() -> Mapping[str, Optional[str]]:
         # FIXME: Add tooling to aid in prefix choice
         #        https://github.com/DataBiosphere/azul/issues/3027
         'AZUL_TDR_SOURCES': ','.join([
-            'tdr:broad-jade-dev-data:snapshot/hca_dev_20201203___20210426:42',
+            'tdr:broad-jade-dev-data:snapshot/hca_dev_20201203___20210426:42:2',
         ]),
         **{
             f'AZUL_TDR_{catalog.upper()}_SOURCES': ','.join([
-                f'tdr:broad-jade-dev-data:snapshot/hca_dev_20201023_ebiv4___20210302:4'
+                f'tdr:broad-jade-dev-data:snapshot/hca_dev_20201023_ebiv4___20210302:4:2'
```

Comment:
IIRC, the prefix `4` gives 200 or so bundles so a partition prefix of two digits doesn't really make sense.